src/relay/backend/aot_executor_codegen.cc

```diff
@@ -583,8 +583,16 @@ class AOTExecutorCodegen : public MixedModeVisitor {
     // performing the preexisting AOT executor code generation phase.
     IRModule mod = IRModule::FromExpr(func);
 
+    backend::FunctionInfo func_info;
+
+    if (memory_plan.defined()) {
+      // TODO(@electriclilies, @jroesch): remove UpdateMainWorkspaceSize
+      func_info = tec::UpdateMainWorkspaceSize(mod, targets_, memory_plan->expr_to_storage_info);
+      mod = WithAttr(mod, "main_func_info", func_info);
+    }
```

Comment:
Can we condense this to:
```suggestion
    if (memory_plan.defined()) {
      // TODO(@electriclilies, @jroesch): remove UpdateMainWorkspaceSize
      backend::FunctionInfo func_info = tec::UpdateMainWorkspaceSize(mod, targets_, memory_plan->expr_to_storage_info);
      mod = WithAttr(mod, "main_func_info", func_info);
    }
```